SU2_CFD/include/variables/CTurbSAVariable.hpp

```diff
@@ -87,4 +75,26 @@ class CTurbSAVariable final : public CTurbVariable {
    */
   inline su2double GetVortex_Tilting(unsigned long iPoint) const override { return Vortex_Tilting(iPoint); }
 
+  /*!
+   * \brief Get the value of the turbulence kinetic energy.
+   * \return the value of the turbulence kinetic energy.
+   */
+  inline su2double GetSSTVariables_k(unsigned long iPoint) const { return k(iPoint); }
+
+  /*!
+   * \brief Get the value of the turbulence frequency Omega.
+   * \return the value of the turbulence frequency Omega.
+   */
+  inline su2double GetSSTVariables_omega(unsigned long iPoint) const { return Omega(iPoint); }
+
+  /*!
+   * \brief Set the value of the SST variables computed with SA solution.
+   * \param[in] val_k
+   * \param[in] val_Omega
+   */
+  void SetSSTVariables(unsigned long iPoint, su2double val_k, su2double val_Omega) {
+    k(iPoint) = val_k;
```

Comment:
Are these changes related to the main theme? or were just debugging? If they can be separated to another PR let's do that please.

Reply:
These are for computing the grid size for scale resolving simulations from an SA solution. Probably it has to be revised a little bit, since I found out that the formulation for the TKE is different if the QCR mod is used, thus I think I'll remove them for now.